catalog/spatial/geocoding/spatial-geocoding-index/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesLuceneIndexer.java

```diff
@@ -149,8 +150,15 @@ public void updateProgress(final int progress) {
 
     IndexWriter createIndexWriter(final boolean create, final Directory directory)
             throws IOException {
+
         final IndexWriterConfig indexWriterConfig = new IndexWriterConfig(ANALYZER);
-        indexWriterConfig.setOpenMode(create ? OpenMode.CREATE : OpenMode.APPEND);
+
+        // Set to CREATE mode if the index does not exist.
+        if(!DirectoryReader.indexExists(directory)) {
+            indexWriterConfig.setOpenMode(OpenMode.CREATE);
+        } else {
+            indexWriterConfig.setOpenMode(create ? OpenMode.CREATE : OpenMode.APPEND);
+        }
```

Comment:
This change sets mode to CREATE when there is no existing index.  This eliminates the need for the user to specify true for the create boolean on the first invocation of createIndex within the GeoNamesIndexer.